blocks/callout/callout.css

```diff
@@ -0,0 +1,96 @@
+.callout {
+  > div {
+    display: flex;
+    flex-direction: column;
+    background: var(--theme-clr-background);
+    height: 42.625rem;
+  }
+
+  .callout-image {
+    flex: 1;
+    width: 100%;
+    height: 50%;
+
+    picture, picture img {
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
+  }
+
+  .callout-content {
+    height: 50%;
+    color: var(--theme-clr-text);
+    padding: 5rem .9375rem;
+
+    h3 {
+      font-family: MiloPro-Bold, Helvetica, sans-serif;
+      font-size: var(--heading-font-size-m);
+      letter-spacing: -.03125rem;
+      margin: 0 0 1rem;
+    }
+
+    p {
+      font-family: OpenSans, Helvetica, sans-serif;
+      font-size: 1rem;
+      line-height: 1.5rem;
+      letter-spacing: .015625rem;
+    }
+
+    .button-container {
+      margin: 0;
+      > a {
+        display: inline-block;
+        margin-top: 1.5rem;
+        min-width: 15.625rem;
+        padding: .875rem;
+        background: 0 0;
+        border: 2px solid #fff;
+        border-radius: .125rem;
+        font-family: MiloPro-Bold, Helvetica, sans-serif;
+        font-size: 1rem;
+        text-align: center;
+        line-height: 1.5rem;
+        letter-spacing: .03125rem;
+        color: #fff;
+      }
+    }
+  }
+}
+
+.callout-wrapper {
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0 !important;
+  margin-bottom: 3.125rem !important;
+}
+
+@media (min-width: 67.5625rem) {
```

Comment:
let's use media breakpoints as mentioned [here](https://github.com/aemsites/ingredion/issues/29#issuecomment-2492960849).

Let's use a constant number instead of rem since it depends on other factors.